Automattic/jetpack
# Add ability to run block tests against specific Gutenberg versions

#### Changes proposed in this Pull Request:
* Adds a script to allow running of block unit and fixture tests against specific versions of gutenberg

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Checkout PR  
* Run `pnpm fixtures:test:gbv trunk 3774b1489f17de1da53a96a2066ce9a8f996293c`
* This will check out the commit that caused the recent 10.9.0 regressions with the subscriptions block, so once this task has completed you should see 2 block fixture failures
* Run `pnpm fixtures:test:gbv test-validation` to just run the tests again
* Run `pnpm fixtures:test:gbv reset` and check that `./packages` and `./tests/temp-gutenberg-checkout` dirs have been removed and the `./package.json` changes have been reverted and `pnpm test-extensions` completes successfully
* Run `pnpm fixtures:test:gbv trunk` and make sure all the tests complete successfully

#### Todo:

- [ ] Get block unit tests running
